src/Rasterization/Renderers/SVGRenderer.php

```diff
@@ -91,7 +91,7 @@ private static function prepareColor($color, SVGNode $context)
      */
     private static function getNodeOpacity(SVGNode $node)
     {
-        $opacity = $node->getStyle('opacity');
+        $opacity = $node->getComputedStyle('opacity');
```

Comment:
This is wrong. Opacity is not inherited in CSS, but using `getComputedStyle()` would make it be inherited.